### PR TITLE
Add close application operation

### PR DESCRIPTION
**Pull request by arthurashman**
## Description of change
Adds an operation to mark an application as complete via a new {id}/complete endpoint
Updates review_status

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-148

### CODE REVIEW

File: app/api/datastore/v2/reviewing.rb [hunk 1]
```diff
@@ -23,6 +22,21 @@ class Reviewing < Base
...
+
+        desc 'Mark an application as complete.'
+        params do
+          requires :application_id, type: String, desc: 'Crime Application UUID'
+        end
+
+        route_param :application_id do
+          resource :complete do
+            put do
+              complete_params = declared(params).symbolize_keys
```
Comment (zheileman) [inline app/api/datastore/v2/reviewing.rb:34]: If the only param in this endpoint is `application_id` we probably don't need this, and we can replace it with just `params[:application_id]`
Unless you are expecting some additional params will be added in another PR, then it makes sense 👍

**Review by zheileman — APPROVED** [review]

**Review by timpeat — APPROVED** [review]
LGTM! 👍
